lib/node/runner/node_runner_test.go

```diff
@@ -238,29 +238,6 @@ func TestCreateNodeRunner(t *testing.T) {
 	require.Equal(t, 3, len(nodeRunners))
 }
 
-// Check that when a node receives transaction, the node broadcasts it to validators
-func TestNodeRunnerTransactionBroadcast(t *testing.T) {
```

Comment:
Why is this removed ?

Reply:
@Geod24 This test does not test to broadcast Transaction. As you see, it checks it's own `TransactionPool`. We need new test code for broadcasting.